src/parser/jobs/mnk/index.tsx

```diff
@@ -21,25 +21,56 @@ export default new Meta({
 		<Message warning icon>
 			<Icon name="warning sign"/>
 			<Message.Content>
-				<Trans id="mnk.about.description.warning"><b>The module is still a work in progress</b> and may occasionally give you bad feedback. If you notice any issues, or have any questions or feedback, please drop by our Discord channel!</Trans>
+				<Trans id="mnk.about.description.warning">
```

Comment:
Consider using markdown for this, given you're using some formatting tags.

Reply:
I feex